README.md

```diff
@@ -7,15 +7,37 @@
 
 [bioconda-badge-link]: https://img.shields.io/conda/dn/bioconda/fgsv.svg?label=Bioconda
 [bioconda-link]:       http://bioconda.github.io/recipes/fgsv/README.html
-[github-badge]:        https://github.com/fulcrumgenomics/fgsv/actions/workflows/unittests.yaml/badge.svg
+[github-badge]:        https://github.com/fulcrumgenomics/fgsv/actions/workflows/unittests.yaml/badge.svg?branch=main
 [github-link]:         https://github.com/fulcrumgenomics/fgsv/actions/workflows/unittests.yaml
 [scala-badge]:         https://img.shields.io/badge/language-scala-c22d40.svg
 [scala-link]:          https://www.scala-lang.org/
 [license-badge]:       https://img.shields.io/badge/license-MIT-blue.svg
 [license-link]:        https://github.com/fulcrumgenomics/fgsv/blob/main/LICENSE
 
-Tools to find evidence for structural variation.
+Tools for calling breakpoints and exploring structural variation.
 
 ## Documentation
 
-Documentation can be found in the [docs folder](docs/01_Introduction.md)
+Documentation can be found in the [docs folder](docs/01_Introduction.md).
+
+## Introduction to the `fgsv` Toolkit
+
+The tool [`fgsv SvPileup`](https://github.com/fulcrumgenomics/fgsv/blob/main/docs/tools/SvPileup.md) takes a query-grouped BAM file as input and scans through each template one at a time, where a template is the full collection of reads and alignments from a single source molecule.
+Primary and supplemental alignments for a template are used to construct a “chain” of aligned sub-segments in a way that is order and strand-aware.
+These aligned sub-segments relate to each other through typical alignment mechanisms like insertions and deletions but also contain information about the relative orientation of the sub-segment to the reference genome and importantly, jumps between reference sequences (chromosomes).
```

Comment:
1. This was confusing, since most folks think of indels _within_ an alignment (sub-segment), versus here where we're thinking _between_ sub-segments.  
2. Should we inversions, inter and intra contig translocations, not just the latter?